web/ui/.nvmrc

```diff
@@ -1 +1 @@
-v20.5.1
+v22.21.1
```

Comment:
For the record, this version needs to be sync with the docker image used to build Prometheus: https://github.com/prometheus/golang-builder/blob/master/1.25/base/Dockerfile#L30.

The good thing is, it is already at the version 22.
So all good.